### PR TITLE
[Fix]Events were sorted in ascending order

### PR DESCRIPTION
**Pull request by arghya17**
I used sort() function and isBefore() method to compare from dayjs module to sort the events in ascending order.

### CODE REVIEW

File: src/logic/events.ts [hunk 1]
```diff
@@ -6,7 +6,17 @@ const DATE_FORMAT = 'DD MMM, YYYY hh:mm A'
 export const getEvents = async(): Promise<any> => {
   const { data } = await axios.get(`https://www.googleapis.com/calendar/v3/calendars/5rbvb4k7c10ujfrpeboh5je074@group.calendar.google.com/events?key=${import.meta.env.VITE_CALENDAR_KEY}`)
   const { items } = data;
-  return items.filter((i: any) => dayjs(i.end.dateTime).isAfter(dayjs()) && !i.recurrence)
+  const arr = items.filter(
```
Comment (arnabsen1729) [inline src/logic/events.ts]: change variable names `arr` to `sortedEvents`

**Comment by arnabsen1729** [comment]
Fixes #32

File: src/logic/events.ts [hunk 2]
```diff
@@ -6,7 +6,17 @@ const DATE_FORMAT = 'DD MMM, YYYY hh:mm A'
...
+    (i: any) => dayjs(i.end.dateTime).isAfter(dayjs()) && !i.recurrence
+  );
+  arr.sort((a: any, b: any) =>
+    dayjs(a.end.dateTime).isBefore(dayjs(b.end.dateTime))
+      ? -1
+      : dayjs(b.end.dateTime).isBefore(dayjs(a.end.dateTime))
+      ? 1
+      : 0
```
Comment (raydwaipayan) [inline src/logic/events.ts]: This logic is redundant.

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description

**Review by raydwaipayan — CHANGES_REQUESTED** [review]

**Comment by arghya17** [comment]
DeepinScreenshot_20210718203812
Fixed issue screen shot

**Comment by arnabsen1729** [comment]
LGTM :+1:

**Review by arnabsen1729 — APPROVED** [review]

**Review by raydwaipayan — APPROVED** [review]
LGTM

**Pull request merged**